src/Files.App/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -530,14 +587,19 @@
 				LabelPosition="Collapsed"
 				Style="{StaticResource ToolBarAppBarButtonFlyoutStyle}"
 				ToolTipService.ToolTip="{helpers:ResourceString Name=Layout}">
+
 				<AppBarButton.Icon>
-					<FontIcon FontSize="10" Glyph="&#xE152;" />
```

Comment:
Is there any visual change?

Reply:
No, because default is 10

Before/After
![image](https://user-images.githubusercontent.com/62196528/234301730-124376f6-71ea-40e5-9084-96bc3eefbd8b.png)